data/json/harvest.json

```diff
@@ -2,16 +2,27 @@
   {
     "id": "null",
     "type": "harvest",
-    "entries": []
+    "entries": [  ]
   },
   {
     "id": "fish_small",
     "type": "harvest",
-    "entries": [
-      { "drop": "fish", "scale_num": [ 0.2, 0.4 ], "max": 2 },
-      { "drop": "offal", "base_num": [ 0, 1 ], "scale_num": [ 0.2, 0.4 ], "max": 1 },
-      { "drop": "stomach" },
-      { "drop": "bone", "base_num": [ 1, 2 ], "scale_num": [ 0.4, 0.5 ], "max": 4 }
-    ]
+    "entries": [ {
+      "drop": "fish",
+      "scale_num": [ 0.2, 0.4 ],
+      "max": 2
+    }, {
```

Comment:
This needs an indent. It looks like it belongs to the main object, not `entries`.

Reply:
Agreed, that's a bug.

Reply:
Hmm, this is a lot trickier to address than expected.  I think the best we can do is:
```
"entries": [ {
    "drop": "fish",
    "scale_num": [ 0.2, 0.4 ],
    "max": 2
  }
]
```